Userland/Libraries/LibSyntax/Highlighter.cpp

```diff
@@ -168,6 +170,8 @@ StringView language_to_string(Language language)
 StringView common_language_extension(Language language)
 {
     switch (language) {
+    case Language::CMake:
+        return "cmake"sv;
```

Comment:
Do we know how to tell the Syntax engine that "this specific filename is probably $language" ? If not, probably worth an enhancement for CMakeLists.txt, CMakeCache.txt, build.ninja, Makefile, etc .